frontend/src/pages/student/Facharbeit/FacharbeitApplicationForm.tsx

```diff
@@ -97,41 +120,23 @@ function FacharbeitApplicationForm() {
         <div id="teacherInputsAndButtons" style={{ display: 'flex', flexDirection: 'column', width: '40%' }}>
           <h2> </h2>
           <FormControl style={{ width: '100%', marginTop: '32px' }} variant="outlined">
-            <InputLabel>Fach</InputLabel>
-            <Select
-              label="Fach"
-              onChange={handleSubjectInputChange}
-            >
-              {subjectList.map((subject) => (
-                <MenuItem value={subject.id}>{subject.name}</MenuItem>
-              ))}
-            </Select>
+            <TextField label="Fach" variant="outlined" style={{ width: '100%', minWidth: '65px' }} onChange={handleSubjectInputChange} />
           </FormControl>
           <FormControl style={{ width: '100%', marginTop: '3vh' }} variant="outlined">
-            <InputLabel>Betreuende Lehrkraft</InputLabel>
-            <Select
-              label="Betreuende Lehrkraft"
-              onChange={handleBetreuendeLehrkraftInputChange}
-            >
-              {betreuendeLehrkraefteList.map((lehrkraft) => (
-                <MenuItem value={lehrkraft.id}>{lehrkraft.name}</MenuItem>
-              ))}
-            </Select>
+            <TextField label="Betreuende Lehrkraft" variant="outlined" style={{ width: '100%', minWidth: '65px' }} onChange={handleBetreuendeLehrkraftInputChange} />
           </FormControl>
           <FormControl style={{ width: '100%', marginTop: '3vh' }} variant="outlined">
-            <InputLabel>Unterrichtende Lehrkraft</InputLabel>
-            <Select
-              label="Unterrichtende Lehrkraft"
-              onChange={handleUnterrichtendeLehrkraftInputChange}
-            >
-              {unterrichtendeLehrkraftList.map((lehrkraft) => (
-                <MenuItem value={lehrkraft.id}>{lehrkraft.name}</MenuItem>
-              ))}
-            </Select>
+            <TextField label="Unterrichtende Lehrkraft" variant="outlined" style={{ width: '100%', minWidth: '65px' }} onChange={handleUnterrichtendeLehrkraftInputChange} />
           </FormControl>
-          <Button style={{ marginTop: '3vh' }} variant="contained" color="primary" onClick={handleSubmitFacharbeit}>
-            Facharbeit einreichen / ändern
-          </Button>
+          {formStatus === 3 ? 
+            <Button style={{ marginTop: '3vh' }} variant="contained" color="primary" onClick={handleSubmitAdminFacharbeit}>
+              Facharbeit ändern
+            </Button>
+          :
+            <Button style={{ marginTop: '3vh' }} variant="contained" color="primary" onClick={handleSubmitFacharbeit}>
+              Facharbeit einreichen / ändern
+            </Button>
+          }
```

Comment:
Nur so als Anmerkung, musst du nicht umsetzen:
Aber du hättest auch einen Button nehmen können und die beschriftung und on ClickMethode mit ternären ausdrücken abhängig vom formStatus machen können:
{formStatus === 3 ? 'FAcharbeit ändern' : 'Facharbeit einreichen'} und analog in der onClick mit der handlerFunction